src/service.rs

```diff
@@ -25,6 +25,7 @@ use crate::{double_nul_terminated, Error};
 
 bitflags::bitflags! {
     /// Enum describing the types of Windows services.
+    #[derive(Debug, Clone, Hash, Eq, PartialEq, Copy)]
```

Comment:
Could you derive `PartialOrd`/`Ord` here to prevent breaking changes?

```rust
#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Clone, Copy)]
```